nixos/modules/system/boot/stage-2-init.sh

```diff
@@ -82,6 +84,9 @@ for o in $(cat /proc/cmdline); do
             set -- $(IFS==; echo $o)
             resumeDevice=$2
             ;;
+        init2=*)
+            set -- $(IFS==; echo $o)
+            initDaemon=$2
```

Comment:
Indeed -- at first I've envisioned that one would just specify `init2=systemd-bootchart` but this does require a NixOS module anyway, so -- no benefits now. I'll change it, thanks!